spec/asyncapi.md

```diff
@@ -1822,7 +1824,8 @@ properties:
     type: string
 required:
 - name
-example:
+examples:
+-
   name: Puma
   id: 1
```

Comment:
Please add `-` and first field in this same level (line):

```suggestion
- name: Puma
  id: 1
```

Reply:
The YAML spec suggests otherwise: https://yaml.org/spec/1.2.2/#example-sequence-of-mappings

Reply:
And in the next examples you have this :)
![image](https://user-images.githubusercontent.com/20404945/151350325-a296ccaa-521a-4dd3-94c1-08027c2e9db3.png)